Docker_Weaviate.ipynb

```diff
@@ -0,0 +1,236 @@
+   "metadata": {},
+   "source": [
+    "In the following cells we load the locally stored data (in json format) and create a function definition for an add_podcast object. \n",
+    "\n",
+    "The name of the object represents the highest level classification for your data, indicated below as podcast_object (in dictionary type). Target class represents the next level in the classification of your data. Here we indicate it below as the string \"Podcast\", but note that multiple classes could have been specified, for example, if we had different categories of podcasts, such as English, Spanish, etc.\n",
+    "\n",
+    "The function definition below is implementing batch_size=1. Note that with larger amounts of data you will want to adjust this setting. Per the documentation: \"batch imports are used to maximize import speed and minimize network latency. Batch import processes multiple objects per request, and clients can parallelize the process.\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open(\"/Users/tdubon/DEMO-semantic-search-podcast/data/podcast_ds.json\", 'r') as f:\n",
+    "    datastore = json.load(f)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(json.dumps(datastore, indent=2))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the cell below we define the batch and the uuid.\n",
+    "\n",
+    "Batch definition is helpful because it's \"a way of importing/creating objects and references in bulk using a single API request to the Weaviate server.\" "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def add_podcasts(batch_size = 1):\n",
+    "    client.batch.configure(batch_size=1)\n",
+    "    with client.batch as batch:\n",
+    "        for i, d in enumerate(datastore):\n",
+    "            print(f\"importing podcast: {i+1}\")\n",
+    "            properties = {\n",
+    "                \"title\": d[\"title\"],\n",
+    "                \"transcript\": d[\"transcript\"]\n",
+    "            }\n",
+    "            podcast_uuid = generate_uuid5('podcast', d[\"title\"] + d[\"transcript\"])\n",
+    "        \n",
+    "            batch.add_data_object(\n",
+    "                data_object=properties, \n",
+    "                class_name= \"Podcast\")\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "add_podcasts(1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next you implement the pipeline and query your data, such as semantic search, generative search, question/answering. In this example we use nearText with the module text2vec-openai which implments text-embedding-ada-002. "
```

Comment:
It would be great if you could create a file called "query.py" and add this part there.